packages/wasm-transformer/README.md

```diff
@@ -30,46 +28,63 @@ This project depends on [wasmparser](https://github.com/yurydelendik/wasmparser.
 
 ## Installation
 
-### Rust
-
-```
-# Cargo.toml
-[dependencies]
-wasm_transformer = "LATEST_VERSION_HERE"
-```
-
-### Javascript
-
 ```
 npm install --save @wasmer/wasm-transformer
 ```
 
 ## Quick Start
 
-### Rust
+For a larger example, see the [wasm-terminal](../../packages/wasm-terminal) package.
 
-For a larger example, see the simple [wasm_transformer_cli](../../examples/wasm_transformer_cli).
+### Node Usage
 
-```rust
-use wasm_transformer::*;
+```js
+const wasmTransformer = require("@wasmer/wasm-transformer");
 
-// Some Code here
+// Read in the input Wasm file
+const wasmBuffer = fs.readFileSync("./my-wasm-file.wasm");
 
-// Read in a Wasm file as a Vec<u8>
-let mut Wasm = fs::read(wasm_file_path).unwrap();
-// Add trampoline functions to lower the i64 imports in the Wasm file
-let lowered_wasm = wasm_transformer::lower_i64_imports(wasm);
-// Write back out the new Wasm file
-fs::write("./out.wasm", &lowered_wasm).expect("Unable to write file");
+// Transform the binary
+const wasmBinary = new Uint8Array(wasmBuffer);
+const loweredBinary = wasmTransformer.lowerI64Imports(wasmBinary);
+
+// Do something with loweredBinary
 ```
 
-### Javascript
+### Inlined (Development) Browser Usage
 
-For a larger example, see the [wasm-terminal](../../packages/wasm-terminal) package.
+The default import of `@wasmer/wasm-transformer` points to the inlined development bundle. This bundle **has it's wasm file from the `wasm_transformer` crate as a Base64 encoded URL in the bundle.** This is done for convience and developer experience. **Production applications that are being used by every day users should be using the production bundles instead**.
```

Comment:
I don't think inlining can be considered only as "development mode".

Rather to point to development vs prod. I think it will be better to do the distinction between Inlined Wasm (default) and non-inlined:

```suggestion
The default import of `@wasmer/wasm-transformer` points to the inlined bundle. This bundle **has the wasm file from the `wasm_transformer` crate as a Base64 encoded URL in the bundle.**
This is done for convenience and developer experience.

However, there are use cases where we you might don't want to use the inlined Wasm (for example, when working with [PWAs](https://developers.google.com/web/progressive-web-apps)) For that case, you should be using the `@wasmer/wasm-transformed/non-inline` version.
```